<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SimpleScripts</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Kosou.ico</normaloff>
   </iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="centralLayout">
    <item>
     <widget class="QWidget" name="mainMenuWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>15</number>
       </property>
       <property name="topMargin">
        <number>15</number>
       </property>
       <property name="rightMargin">
        <number>15</number>
       </property>
       <property name="bottomMargin">
        <number>15</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="sizeConstraint">
          <enum>QLayout::SetNoConstraint</enum>
         </property>

         <item>

          <widget class="QPushButton" name="pushButton_1">
           <property name="text">
            <string>Удаление дубликатов</string>
           </property>
          </widget>

         </item>
         <item>

          <widget class="QPushButton" name="pushButton_2">
           <property name="text">
            <string>Фильтрация двух пачек данных</string>
           </property>
          </widget>

         </item>
         <item>

          <widget class="QPushButton" name="pushButton_3">
           <property name="text">
            <string>Работа с префиксами</string>
           </property>
          </widget>

         </item>
         <item>

          <widget class="QPushButton" name="pushButton_4">
           <property name="text">
            <string>Формирование номеров для Aster</string>
           </property>
          </widget>

         </item>
         <item>

          <widget class="QPushButton" name="pushButton_5">
           <property name="text">
            <string>Форматирование списаний по номерам</string>
           </property>
          </widget>

         </item>
         <item>

          <widget class="QPushButton" name="pushButton_6">
           <property name="text">
            <string>Формирование MYSQL запросов для номеров</string>
           </property>
          </widget>

         </item>
         <item>

          <widget class="QPushButton" name="pushButton_7">
           <property name="text">
            <string>Обработка прайсов Билайн</string>
           </property>
          </widget>

         </item>
         <item>

          <widget class="QPushButton" name="pushButton_8">
           <property name="text">
            <string>Разделение номеров по операторам (подкодам сетей)</string>
           </property>
          </widget>

         </item>
         <item>

          <widget class="QPushButton" name="pushButton_10">
           <property name="text">
            <string>Коды стран (Обработка массы номеров и быстрый поиск)</string>
           </property>
          </widget>

         </item>
         <item>

          <widget class="QPushButton" name="pushButton_9">
           <property name="text">
            <string>О программе</string>
           </property>
          </widget>

         </item>
         
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <!-- Здесь программно добавляйте QStackedWidget для скриптов -->
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
